cockroachdb/docker-compose.cockroachdb.yml

```diff
@@ -0,0 +1,207 @@
+version: "3"
+
+services:
+    acra-keymaker_writer:
+        image: "cossacklabs/acra-keymaker:${ACRA_DOCKER_IMAGE_TAG:-0.93.0}"
+        network_mode: "none"
+        environment:
+            ACRA_MASTER_KEY: ${ACRA_SERVER_MASTER_KEY:-t14/alGiPpS5mOH5R2zOQ8gjUcSQ3b+6l0qD31V+yZE=}
+        volumes:
+            - ./.acrakeys:/keys
+            - ../_common/cockroachdb/certs/:/certs
+        command: >-
+            --client_id=""
+            --tls_cert=/certs/client.root.crt
+            --generate_acrawriter_keys
```

Comment:
lets leave comment that these keys for client-side encryption support because we will place python scripts that support both: transparent and client-side encryption

Reply:
if we are using transparent encryption, i suggest to remove generating these keys completely

Reply:
>  there is no negatives in this way. 

actually, there are. people use our examples to understand how easy it is to use/integrate Acra. Examples should contain MVP, without extra "you don't need these features but here they are".

> we don't focus or mention client-side encryption

i think that client-side encryption examples should be different from server-side encryption examples. 

why generating client-side keys for server-side encryption? how to explain users "oh yes we have this in example but you don't actually need it"? All extra info makes examples more complicated to understand -> people will either ask Qs OR go away frustrated.

Reply:
>  to understand how easy it is to use/integrate Acra

it is only one reason why they open our examples. the next one to see what acra can do, what kind of features it have. we can make focus on some of them and let them try by themselves other features and simplify it. the other one is which features are under control of clients and which controls are tweakable. the other one which controls they can override or change.

anyway, generation of these keys turns on acrastruct's support on the server's side and trying different crypto envelopes.